velox/exec/Window.cpp

```diff
@@ -45,7 +45,14 @@ Window::Window(
         windowNode, pool(), spillConfig, &nonReclaimableSection_);
   } else {
     windowBuild_ = std::make_unique<SortWindowBuild>(
-        windowNode, pool(), spillConfig, &nonReclaimableSection_, &spillStats_);
+        windowNode,
+        pool(),
+        common::PrefixSortConfig{
```

Comment:
Can we make window test a parameterized test with one mode having very large normalized key which allows prefix sort if key column types allowed, or a very small normalized key size like 1 to disable it? So it auto-cover the two code paths in window unit test. The same for window fuzzer test to make sure both code path are covered. Thanks!

Reply:
Window operator has very exhaustive tests for all functions : 

- [RankTest](https://github.com/facebookincubator/velox/blob/main/velox/functions/prestosql/window/tests/RankTest.cpp), 
- [NthValueTest](https://github.com/facebookincubator/velox/blob/main/velox/functions/prestosql/window/tests/NthValueTest.cpp)  
- [AggregateWindowTest](https://github.com/facebookincubator/velox/blob/main/velox/functions/prestosql/window/tests/AggregateWindowTest.cpp).

In these tests we have covered a combination of several [over clauses] (https://github.com/facebookincubator/velox/blob/main/velox/functions/lib/window/tests/WindowTestBase.h#L33)
for each function with different [frames](https://github.com/facebookincubator/velox/blob/main/velox/functions/lib/window/tests/WindowTestBase.h#L54) and multiple data generation strategies (all rows in single partition, all partitions with single row, equally distributed data, randomly generated data).

All these tests have atleast 4 columns between the partition by and order by which should test prefix sort well. 
The Window Fuzzer too generates multiple keys so prefix sort is well tested. But I enhanced it to generate even wider set of keys randomly to add pressure to Prefix sort runs.

For non-prefix sort situations with just one partition column its not very easy to test different combinations of window functions as then you require partitions with exactly one row to get unique results. Infact the WindowFuzzer always generates a row number and sorts by it to get comparable results. 
I added a test in AggregateWindowTest with just a single varchar partition column to test the non prefix-sort case.